### ISSUE TITLE
desktop/mobile notifications

### ISSUE THREAD

**Issue by caryoscelus** [zeronet-conservancy/zeronet-conservancy#41]
the only reason it's hard to implement is because of sandboxing . that shouldn't stop us from trying though

**Comment by slrslr** [comment]
ZeroMail notification is missing, yet that is different issue i guess.

**Comment by caryoscelus** [comment]
@slrslr yep . it's related to how 0mail isn't really integrated into 0net db system because of additional encryption/decryption that is implemented on "client side"

**Comment by caryoscelus** [comment]
turns out undocumented js api for this have been around for a long time : https://github.com/zeronet-conservancy/zeronet-conservancy/blob/d9fea20e62871fffe11ae2196a24d940f7fce199/src/Ui/media/all.js#L1198

still need to put it to use though
